src/XMTP.types.ts

```diff
@@ -38,7 +42,7 @@ export type RemoteAttachmentMetadata = {
 };
 
 export type EncryptedLocalAttachment = {
-  encryptedLocalFileUri: `file://${string}`;
```

Comment:
Probably should also note this as potentially a breaking change?

Reply:
I don't think this should be a breaking change, it's just changing the typescript assertion to not include the templated string because it didn't seem to be working. Either way this property is still just a string though.